internal/servegit/serve.go

```diff
@@ -62,6 +62,7 @@ var indexHTML = template.Must(template.New("").Parse(`<html>
 type Repo struct {
 	Name string
 	URI  string
+	Bare bool
```

Comment:
Related to the changes [here](https://github.com/sourcegraph/sourcegraph/pull/40170)

Reply:
I'm not sure about this, commented here https://github.com/sourcegraph/sourcegraph/pull/40170/files#r952907457